<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestRenderServer</class>
 <widget class="QWidget" name="TestRenderServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="folderNameLabel">
       <property name="text">
        <string>Folder Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameFolderLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="userNameLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="uploadFolderPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Upload Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete Folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="sceneLabel">
       <property name="text">
        <string>Scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sceneLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="renderPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ask Rendering</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
